src/layer/vector/Canvas.js

```diff
@@ -186,9 +186,13 @@ export var Canvas = Renderer.extend({
 		if (typeof layer.options.dashArray === 'string') {
 			var parts = layer.options.dashArray.split(/[, ]+/),
 			    dashArray = [],
+			    dashValue,
 			    i;
 			for (i = 0; i < parts.length; i++) {
-				dashArray.push(Number(parts[i]));
+				dashValue = Number(parts[i]);
+				// Ignore dash array containing invalid lengths
+				if (isNaN(dashValue)) { return; }
+				dashArray.push();
```

Comment:
```suggestion
				dashArray.push(dashValue);
```

Reply:
Woha! Good thing someone is reviewing my crap!